cloudshell-orch-core/cloudshell/workflow/orchestration/app.py

```diff
@@ -2,22 +2,38 @@
 from cloudshell.api.cloudshell_api import ReservedResourceInfo
 from cloudshell.api.cloudshell_api import ReservationAppResource
 
+
 class AppResourceRequestInfo(object):
     def __init__(self, app_resource):
         self.app_resource = app_resource
         """:type : ReservationAppResource"""
-        self.appConfiguration = []
+        self.scripts = {}
+        """:type : Dictionary[str,ScriptRequestInfo]"""
+
+    def add_app_config_param(self, script_alias, key, value):
+        if script_alias not in self.scripts:
+            self.scripts[script_alias] = ScriptRequestInfo(script_alias)
+        self.scripts[script_alias].add_script_config_param(key, value)
```

Comment:
I think this changes the old behavior a bit.
in the previous implementation if someone called set_config_param it would just append all parameters, now it will override all other configs for that same alias.
I personally prefer your implementation, but maybe we need to check the previous behavior?

Reply:
Actually I don't see where do I override something here.

Reply:
yeah you're correct, sorry :X